src/ui/Overlay.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import styles from './styles/overlay.module.css';
+import CharacterSheet from './sheet/CharacterSheet';
+import ProfileSelection from './profile/ProfileSelection';
+import NestedTab from './tabs/NestedTab';
+
+function Overlay() {
+  const [activeTab, setActiveTab] = useState('Select Character');
+  const [sheet, setSheet] = useState(null);
+
+  const onCharacterReady = (characterSheetPromise) => {
+    characterSheetPromise.then((sheetData) => {
+      setActiveTab('Character Sheet');
+      setSheet(sheetData);
+    });
+  };
+
+  return (
+    <div className={`globalOverlay ${styles.Overlay}`}>
```

Comment:
There's a library that simplifies managing lists of classes called [classnames](https://github.com/JedWatson/classnames).

I don't usually pull in small dependencies but this is a really small one that adds a lot of value.

Reply:
I made sure to avoid dependencies in the initial attempt at the project to avoid pulling in unnecessary libraries.  I'm good with adding this one since over time, I'll likely be adding more and more styles.